Tests/Fixtures/TestProject/App_supportedPlatforms/project.yml

```diff
@@ -0,0 +1,32 @@
+name: MyProject
```

Comment:
Thanks for adding this, but can we integrate with `Tests/Fixtures/TestProject/project.yml` as that's the project that gets built on CI to check the generated project is valid. You can even use an `include` in that spec to copy this stuff in

Reply:
ah ok sure

Reply:
Done